### ISSUE TITLE
Documentation Dir needs a top level index

### ISSUE THREAD

**Issue by tmccormi** [LibreHealthIO/LibreEHR#710]
There is no way to provide a link to all the subdirs in the Documentation/ directory from Setup or the Main Menu.
```
1_Installing     3_Billing         5_Healthcare     9_doc_and_logo_sources  README-phpgacl.md
2_Configuration  4_FeatureManuals  8_ForDevelopers  FAQ.MD

```

**Comment by aethelwulffe** [comment]
Agreed.  Index should hold everything, and basically act as a little 
CRM.  Great idea.  Everything is good.

**Comment by tmccormi** [comment]
Not as easy I I would like because each dir is a random collection of PDF, ODT and MD files, not a coherent set of one thing (like rendered HTML)...   I was hoping to create a automatic indexing tool ...
